src/UniversalFunctions.jl

```diff
@@ -241,31 +303,33 @@ Businger momentum similarity `ϕ_m`.
 """
 @inline function phi(p::BusingerParams, ζ, ::MomentumTransport)
     FT = eltype(ζ)
-    if ζ < 0
-        return _phi_m_unstable(ζ, b_m(p))
-    else
-        return FT(a_m(p)) * ζ + FT(1)
-    end
+    # Ensure invalid branch inputs (ζ >= 0 for unstable, ζ < 0 for stable) don't error
+    # even though their result is discarded by ifelse.
+    return ifelse(
+        ζ < 0,
+        _phi_m_unstable(min(ζ, FT(0)), b_m(p)),
+        FT(a_m(p)) * max(ζ, FT(0)) + FT(1),
+    )
 end
 
 """
     phi(p::BusingerParams, ζ, ::HeatTransport)
 
-Businger heat-gradient similarity `ϕ_h`.
+Businger heat/scalar-gradient similarity `ϕ_h`.
 
 # References
  - Stable (ζ >= 0): Eq. A2 (L >= 0) in Nishizawa & Kitamura (2018).
  - Unstable (ζ < 0): Eq. A2 (L < 0) in Nishizawa & Kitamura (2018).
 """
 @inline function phi(p::BusingerParams, ζ, tt::HeatTransport)
     FT = eltype(ζ)
-    if ζ < 0
-        return _phi_h_unstable(ζ, b_h(p))
-    else
-        _a_h = FT(a_h(p))
-        _π_group = FT(π_group(p, tt))
-        return _a_h * ζ / _π_group + FT(1)
-    end
+    _Pr_0 = FT(Pr_0(p))
+    _a_h = FT(a_h(p))
+    return ifelse(
+        ζ < 0,
+        _Pr_0 * _phi_h_unstable(min(ζ, FT(0)), b_h(p)),
+        _Pr_0 + _a_h * max(ζ, FT(0)),
+    )
```

Comment:
This is different from the paper, I guess this is more correct though?

Reply:
Yes, I made the treatment of Pr_0 uniform across the parameterizations. (The fact that it wasn't was a source of discontinuities before.) I wrote about it in the docs: https://github.com/CliMA/SurfaceFluxes.jl/blob/main/docs/src/UniversalFunctions.md (but see I still need to update the Businger-Dyer formulas there). I'll make it clearer in the docstrings too.